runtime/src/nfts.rs

```diff
@@ -44,6 +48,11 @@ decl_module! {
 
 			let metadata = bundled_hash.as_ref().to_vec();
 			let resource_id = <T as pallet_bridge::Trait>::HashId::get();
+
+			// Burn additional fees
+            let nft_fee = <T as pallet_balances::Trait>::Balance::from(NFT_FEE);
+            <fees::Module<T>>::burn_fee(&who, nft_fee)?;
```

Comment:
As @vedhavyas mentioned above, we should probably give those to the author as well.